app/main.py

```diff
@@ -1,25 +1,68 @@
 from abc import ABC
+from pyclbr import Class
+from typing import Any
 
 
 class IntegerRange:
-    pass
+    def __init__(self, min_amount: int, max_amount: int) -> None:
+        self.min_amount = min_amount
+        self.max_amount = max_amount
+
+    def __set_name__(self, owner: Any, name: str) -> None:
+        self.protected_name = f"_{name}"
+
+    def __get__(self, instance: Any, owner: Any) -> str:
+        return getattr(instance, self.protected_name)
```

Comment:
The `__get__` method in the `IntegerRange` class should return the value as a string, as indicated by its return type annotation `-> str`. Consider converting the returned value to a string.